srw/srw.cpp

```diff
@@ -479,6 +541,7 @@ class srw {
 					}
 
 					stream->write((const char *)sph, total_size(sph) + sizeof(struct sph));
+					stream->close();
```

Comment:
It's request stream. Here we indicate that there will be no more data sent to worker.